catalog/core/catalog-core-versioning/versioning-common/src/test/groovy/ddf/catalog/core/versioning/HistoryMetacardImplSpecTest.groovy

```diff
@@ -101,4 +162,31 @@ class HistoryMetacardImplSpecTest extends Specification {
         res.metacard = metacard
         return res
     }
+
+    def nonBasicMetacard() {
+        def res = [:]
+        res.id = "NonBasicMetacardTypeId"
+        res.title = "NonBasic Title"
+        res.tags = ["NonBasicTags"]
+        res.attributeDescriptor = new AttributeDescriptorImpl(
+                "New Attribute Name",
+                true,
+                true,
+                false,
+                false,
+                BasicTypes.STRING_TYPE)
+        res.metacardType = new MetacardTypeImpl(
+                "NonBasicType",
+                BasicTypes.BASIC_METACARD.attributeDescriptors + [res.attributeDescriptor])
```

Comment:
Could also use the new `MetacardTypeImpl` constructor (which I think makes the intent clearer) 😀